lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/api/VortexFuture.java

```diff
@@ -19,18 +19,28 @@
 package org.apache.reef.vortex.api;
 
 import org.apache.reef.annotations.Unstable;
+import org.apache.reef.wake.EventHandler;
+import org.apache.reef.wake.impl.ThreadPoolStage;
 
 import java.util.concurrent.*;
 
 /**
  * The interface between user code and submitted task.
- * TODO[REEF-505]: Callback features for VortexFuture.
  */
 @Unstable
 public final class VortexFuture<TOutput> implements Future<TOutput> {
   private TOutput userResult;
   private Exception userException;
   private final CountDownLatch countDownLatch = new CountDownLatch(1);
+  private final ThreadPoolStage<TOutput> stage;
+
+  public VortexFuture() {
+    stage = null;
+  }
+
```

Comment:
Since those are public constructors, it would be good to have javadoc description.